packages/vue-email/src/components/e-button.vue

```diff
@@ -1,72 +1,52 @@
 <template>
-  <a
-    v-bind="$props"
-    data-id="__vue-email-button"
-    :href="href"
-    :target="target"
-    :style="styleToString(buttonStyle({ ...style, pX, pY }))"
-  >
-    <span v-html="firstSpan"></span>
-    <span :style="styleToString(buttonTextStyle({ ...style, pX, pY }))">
+  <a data-id="__vue-email-button" :style="buttonStyle(px, py)">
+    <span v-html="firstSpan" />
+    <span :style="buttonTextStyle(py)">
       <slot />
     </span>
-    <span v-html="secondSpan"></span>
+    <span v-html="secondSpan" />
   </a>
 </template>
 
 <script lang="ts" setup>
-import { pxToPt, styleToString } from '../utils';
-import type { LinkHTMLAttributes } from 'vue';
-
-interface Props extends /* @vue-ignore */ Omit<LinkHTMLAttributes, 'style'> {
-  href: string;
-  target?: string;
-  style?: any;
-  pX?: number;
-  pY?: number;
-}
-
-const props = withDefaults(defineProps<Props>(), {
-  href: '',
-  target: '_blank',
-  pX: 0,
-  pY: 0,
+import { pxToPt } from '../utils';
+
+const props = defineProps({
+  px: {
+    type: String,
+    default: '0',
+  },
+  py: {
+    type: String,
+    default: '0',
+  },
 });
 
-const y = props!.pY * 2;
+const y = Number.parseInt(props.py) * 2;
 const textRaise = pxToPt(y.toString());
 
-const buttonStyle = (style?: Record<string, string | number> & { pY?: number; pX?: number }) => {
-  const paddingY = style?.pY || 0;
-  const paddingX = style?.pX || 0;
-
+const buttonStyle = (py: string = '0', px: string = '0') => {
   return {
-    ...style,
     lineHeight: '100%',
     textDecoration: 'none',
     display: 'inline-block',
     maxWidth: '100%',
-    padding: `${paddingY}px ${paddingX}px`,
+    padding: `${py}px ${px}px`,
   };
 };
 
-const buttonTextStyle = (
-  style?: Record<string, string | number | null> & { pY?: number; pX?: number }
-) => {
-  const paddingY = style?.pY || 0;
-
+const buttonTextStyle = (py: string = '0') => {
   return {
```

Comment:
Passing the `style` prop gives the user the ability to add custom styles to the component, particularly when they don't want to use Tailwind classes. What's your point in this case?